app/Comments.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Comments extends Model
+{
+    protected $table = 'comments';
+    const LIMIT_COMMENT = 5;
+    /**
+     * Relationship one to many.
+     *
+     * @return App\Blog
+     */
+    public function blog()
+    {
+        return $this->belongsTo(Blog::class);
+    }
+
+    /**
+     * Relationship many to one.
+     *
+     * @return array App\Blog
```

Comment:
Comment and content does not match each other

Reply:
I fixed